interface/frontend/package.json

```diff
@@ -82,6 +82,8 @@
     "vue-style-loader": "^3.0.1",
     "vue-template-compiler": "^2.4.2",
     "cornerstone-core": " 0.13.0",
+    "cornerstone-tools": " 0.10.0",
+    "jquery": "^3.2.1",
```

Comment:
There was a question-mark in previous PRs regarding needing jQuery - does that apply here? :)

Reply:
I'll be glad to get rid of jquery, but cornerstoneTools simply does not work with out it

Reply:
Sadly that seems to be the case with `cornerstone-tools` so we will have to make do with it. I went through the source of cornerstone-tools and their usage of jQuery is somewhat justifiable for complex input event handling. Had they have a more experienced JS contributor, a couple of addEventListener together with more modular project structure could have avoided jQuery.